app/views/help/cookie_settings.html.erb

```diff
@@ -7,7 +7,9 @@
       aria_live: true,
```

Comment:
For some reason `aria-live="polite"` is not present in the markup despite it being `true` here - and because of this VoiceOver doesn't announce any changes when the form is submitted.

Reply:
Good spot, thanks @injms . Hopefully it just needs a gem bump / rebase 🤞

Reply:
Updated to the latest version of `govuk_publishing_components` which includes that change. Should all be ok now 👍